FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/demos/PersistentBottomSheetActivity.kt

```diff
@@ -240,7 +240,7 @@ class PersistentBottomSheetActivity : DemoActivity(), SheetItem.OnClickListener,
                     ContextCompat.getColor(this, R.color.bottomsheet_horizontal_icon_tint),
```

Comment:
from the screenshot, icons still doesn't seem to be visible in dark mode

Reply:
We are following the central Fluent Theming and colors here, the problem was the consistency of the icon, this one row, in specific, had been different from the rest of the icons. I am attaching more screenshots of the entire bottomDrawer for your reference.
If we want to change the color for these, then that would have to be done for the entire element, the aim for this PR was to make them consistent throughout the UI.

Reply:
I have updated the screenshots, please check now
The perceived lack of visibility may also be due to the fact that the icons that have been updated are set to disabled state, i.e, disabled for user interaction and thus are dimmer than the rest